src/js/components/SelectMultiple/index.d.ts

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { BasicSelectProps } from '../Select';
+
+export interface SelectMultipleProps extends BasicSelectProps {
+  defaultValue?: (string | number | object)[];
+  help?: React.ReactNode;
+  limit?: number;
+  value?: (string | number | object)[];
+  showSelectedInline?: boolean;
+  separateSelected?: boolean;
```

Comment:
`separateSelected` doesn't immediately make sense to me. Thinking out loud with some ideas:
- selectionsTop
- sortSelectedOnClose
- liftSelected
- groupSelected

Also, should the value default to `true` ? or will that cause issues

Reply:
I changed the prop name to `sortSelectedOnClose` and have it default to true

Reply:
My one concern with using "sort" is that if the caller has ordered the options to be: [A, C, B, E, D] and the user selects [C, B], the name "sortSelectedOnClose" implies to me that the component will covert the selected to [B, C].